packages/sitecore-jss-nextjs/src/components/Placeholder.tsx

```diff
@@ -5,8 +5,19 @@ import {
 } from '@sitecore-jss/sitecore-jss-react';
 import { ComponentPropsReactContext } from './ComponentPropsContext';
 
+const PREFIX_PLACEHOLDER = 'container-{*}';
```

Comment:
This logic should be placed to the `sitecore-jss-react` Placeholder, I think...because we already added some SXA related logic there, and it also should be there